tower-layer/src/lib.rs

```diff
@@ -20,7 +20,7 @@
 ///
 /// Take request logging as an example:
 ///
-/// ```rust
+/// ```rust,ignore
```

Comment:
We should probably fix at least the doctests before merging this. Ideally also the tests.